tools/msvs/msi/nodemsi.wixproj

```diff
@@ -54,9 +54,12 @@
     </WixExtension>
   </ItemGroup>
   <ItemGroup>
-    <!-- <EmbeddedResource Include="i18n\de-de.wxl" /> -->
-    <EmbeddedResource Include="i18n\en-us.wxl" />
+    <!--
```

Comment:
@piccoloaiutante I'm a little hesitant to use a weekly build because the WiX website says not to, and breaking the MSI is something that would cause major pain for users. But I'd be willing to give it a try, provided that we build release candidates for a few versions and get them well tested. Using the translations with the release version still sounds better.

Do we have a plan for when to start shipping with this?